bot/cryptocom-ai-telegram-bot/bot.py

```diff
@@ -0,0 +1,135 @@
+import os
```

Comment:
looks good. Could we add some docstrings to the functions?

Reply:
ok